src/status_im/core.cljs

```diff
@@ -40,8 +40,6 @@
 
   (utils.universal-links/initialize)
 
-  (async-storage/get-item :new-ui-enabled? #(reset! config/new-ui-enabled? %))
-
```

Comment:
I think we should not remove storing of this flag. We are opting for enabling new ui by default but if we toggle to the old ui for testing or something, it should not revert automatically on app restart.

Reply:
or are we removing togging functionality from UI completely

Reply:
there is no toggle anymore

Reply:
that's good :D

Reply:
I think we had a concern that the QA team had tests using the old ui. I think we need to discuss these changes with them before merging this.

@qoqobolo, @churik